spec/features/user/user_can_see_github_repos_spec.rb

```diff
@@ -18,7 +18,7 @@
       within('#github') do
         page.assert_selector('.repo', count: 5)
         within(first('.repo')) do
-          expect(page).to have_content('test_user/repo1')
+          expect(page).to have_content('alexander-mathieu/little_shop_v2')
```

Comment:
This will fail if anyone except for you is running the test that's why we mock it out.

Reply:
The cassette will keep all that info in there for the next user. I will create a mock token though.